src/ansi-c/ansi_c_language.cpp

```diff
@@ -118,7 +118,8 @@ bool ansi_c_languaget::typecheck(
     return true;
   }
 
-  remove_internal_symbols(new_symbol_table);
```

Comment:
The norm in the codebase is to pass `message_handlert` around rather than `messaget`. I suppose the idea is perhaps in the future an instance of `messaget` will denote a source of log messages, and a handler the means of printing those messages.